### PR TITLE
Fix usermock

### PR DESCRIPTION
**Pull request by LoganHollins**
Does this seem like a proper mock, or is there something i am missing?

### CODE REVIEW

File: server/mocks/admin.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        'firstName': 'Lou',
+        'lastName': 'Ferrigno',
+        'email': 'lou@email.com',
+        'lastLoginAt': '2016-02-09T23:06:02.053',
+        'createdAt': '2015-12-09T07:00:32.113',
+        'updatedAt': '2015-12-09T07:01:23.92',
+        'roles': [
+          'Admin'
+        ]
+      };
```
Comment (jsedlacek) [inline server/mocks/admin.js]: fix indentation

File: server/mocks/admin.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  router.get('/', function (req, res) {
+    if (req.cookies.user == USER.id) {
+      res.send(USER);
+    } else {
+      res.status(401).send({
+        errors: {base: ['Not logged in']}
+      })
```
Comment (jsedlacek) [inline server/mocks/admin.js]: missing semicolon

**Comment by jsedlacek** [comment]
Why do we have `mocks/admin.js` and `mocks/users.js`? The `admin.js` contains API from `users` namespace.

**Comment by LoganHollins** [comment]
@jsedlacek should i move the admin.js inside the users.js as `router.get('/me', ...)`?

**Comment by jsedlacek** [comment]
@LoganHollins Yes please. It's much easier to find the corresponding routes this way.

**Comment by LoganHollins** [comment]
@jsedlacek I implemented the change you recommended, do you think this will work better now?

**Comment by jsedlacek** [comment]
@LoganHollins can we megre USERS and LOGIN fixtures? They both seem to represent same entity.

**Pull request merged**